packages/boilerplate-generator/boilerplate_web_browser_template.js

```diff
@@ -0,0 +1,76 @@
+// Template function for rendering the boilerplate html for browsers
```

Comment:
Certainly doesn't affect the outcome but I don't see a reason why we would abandon our previous naming convention on these new template files and not keeping the `arch` present in the filename verbatim. (For example, this file might be  `boilerplate_web.cordova_template.js` since the original was `boilerplate_web.cordova.html`).